.env.example

```diff
@@ -0,0 +1,4 @@
+# Config for testing FirestoreAPL, that requires actual connection to Firestore
+TEST_FIRESTORE_PROJECT_ID=
+TEST_FIRESTORE_PRIVATE_KEY=
+TEST_FIRESTORE_CLIENT_EMAIL=
```

Comment:
These are only for test purposes, so marked with TEST_

If they don't exist, tests will be skipped

Reply:
Maybe keeping them in the `.env.test` would be a better place then?